include/vdml/vdml.h

```diff
@@ -32,6 +32,8 @@
  * If a port isn't yet registered, it registered as a motor automatically.
  * If a mutex cannot be taken, errno is set to EACCES (access denied) and
  * returns.
+ * 
+ * FIXME: If this is used in a function that returns a double, PROS_ERR_F should be returned
```

Comment:
Is this necessary to note? Either make versions that return floating point error values, or just leave in the workaround you had below for the gyro.

Reply:
I've changed the message to a simple note that it should only be used by a function that returns `int32_t` to hopefully reduce the chances of someone making that mistake in the future.